api/server.py

```diff
@@ -372,6 +374,49 @@ def delete_all_projects():
     # Number of prizes they can choose per challenge
     # ProjectID that won the challenge
 
+@app.route('/api/import-challenges')
+def import_challenges():
+    devpost_url = "https://bitcamp2018.devpost.com/submissions"
```

Comment:
Let's rename to `DEVPOST_ROOT_URL` and move into `config.py` so that future hackathons don't need to dig through this huge `server.py` file to customize their app, but do everything in a centralized place. Once you move this into `config.py`, you should be able to access via `current_app.config['DEVPOST_ROOT_URL']`